sdk/connectedvmwarevsphere/Azure.ResourceManager.ConnectedVMwarevSphere/tests/tests/Tests/ClusterTests.cs

```diff
@@ -0,0 +1,162 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Resources.Models;
+using Azure.ResourceManager.TestFramework;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using Azure.Core;
+using System.Linq;
+using Azure.ResourceManager.ConnectedVMwarevSphere.Models;
+using Azure.ResourceManager.ConnectedVMwarevSphere.Tests.Helpers;
+
+namespace Azure.ResourceManager.ConnectedVMwarevSphere.Tests.tests.Tests
+{
+    public class ClusterTests : ConnectedVMwareTestBase
+    {
+        public ClusterTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        private async Task<VMwareClusterCollection> GetVMwareClusterCollectionAsync()
+        {
+            var resourceGroup = await CreateResourceGroupAsync();
+            return resourceGroup.GetVMwareClusters();
+        }
+
+        [AsyncOnly]
```

Comment:
Please remove the `AsyncOnly` attribute in all the tests and also record the tests in sync mode, thanks!

Reply:
we need to test both aync and sync

Reply:
You just need to remove the `AsyncOnly` and re-run the tests, our test framework will automatically record the test in sync mode.

Reply:
removed this attribute and I have recorded test in both sync and async mode.